src/Lib/Managers/SelectedBoundManager.vala

```diff
@@ -43,6 +43,13 @@ public class Akira.Lib.Managers.SelectedBoundManager : Object {
     private double initial_width;
     private double initial_height;
 
+    // drag move state
```

Comment:
Better be a bit more explicit with comments.
Something like "Attributes to keep track of the mouse dragging coordiantes."